inst/recorder/www/inject-recorder.js

```diff
@@ -157,9 +157,33 @@ window.recorder = (function() {
                 Shiny.onInputChange("testevents:shinytest.testevents", recorder.testEvents);
             }
 
+            if (message.snapshotKeypress) {
+              triggerSnapshot();
+            }
+
             (function() { eval(message.code); }).call(status);
         });
 
+        var triggerSnapshot = function() {
```

Comment:
Can you change this to `function triggerSnapshot()` just to be consistent with the other code here?